docs/scos/dev/upgradability-services/new-documentation/unique-entity-name-not-unique.md

```diff
@@ -0,0 +1,296 @@
+---
+title: Entity name is not unique
+description: Reference information for evaluator and upgrader tools.
+last_updated: Mar 23, 2022
+template: concept-topic-template
+---
+
+## Entity name is not unique
+
+The names of the following entities should be unique:
```

Comment:
Are you sure that entities is appropriate world for constants and methods?

Reply:
Good point. 
What would you use? 
Names are not unique?